lib/node_modules/@stdlib/math/base/special/modf/manifest.json

```
@@ -0,0 +1,48 @@
+{
+  "options": {},
+  "fields": [
+    {
+      "field": "src",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "include",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "libraries",
+      "resolve": false,
+      "relative": false
+    },
+    {
+      "field": "libpath",
+      "resolve": true,
+      "relative": false
+    }
+  ],
+  "confs": [
+    {
+      "src": [
+        "./src/main.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/math/base/assert/is-infinite",
+        "@stdlib/math/base/assert/is-nan",
+        "@stdlib/number/float64/base/from-words",
+        "@stdlib/number/float32/base/to-word",
+        "@stdlib/constants/float64/pinf",
+        "@stdlib/constants/float64/exponent-bias",
+        "@stdlib/constants/float64/high-word-exponent-mask",
+        "@stdlib/constants/float64/high-word-significand-mask"
```

```suggestion
        "@stdlib/math/base/assert/is-nan",
        "@stdlib/number/float64/base/from-words",
        "@stdlib/number/float64/base/to-words",
        "@stdlib/constants/float64/pinf",
        "@stdlib/constants/float64/exponent-bias",
        "@stdlib/constants/float64/high-word-exponent-mask",
        "@stdlib/constants/float64/high-word-significand-mask"
```

Also, this is not the latest type of manifest that we use, please check